src/components/screens/vacation-requests-screen.tsx

```diff
@@ -73,6 +82,22 @@ const VacationRequestsScreen = () => {
     fetchVacationRequestStatuses();
   }, [vacationRequests]);
 
+    /**
+   * Initialize logged in person data.
+   */
+  const getPersons = async () => {
+    if (!persons.length) {
+      setLoading(true);
+      if (loggedInPerson || config.person.id) setLoading(false);
+      const fetchedPersons = await personsApi.listPersons({ active: true });
+      setPersons(fetchedPersons);
+    }
+  };
+
+  useMemo(() => {
+    getPersons();
+  }, [persons]);
+
```

Comment:
As you said in the now closed comment it looks like this can be removed, I agree, lets remove it